examples/jsm/loaders/OBJLoader.js

```diff
@@ -305,26 +346,45 @@ var OBJLoader = ( function () {
 				this.addVertex( ia, ib, ic );
 				this.addColor( ia, ib, ic );
 
+				// normals
+
+				if ( na !== undefined && na !== '' ) {
```

Comment:
It's probably not worth worrying about  until there's a model that demonstrates the issue but I wonder if it's possible for just _one_ of the vertices in a face to specify a normal while the other two do not.